### ISSUE TITLE
So any new for the SSR support ? 

### ISSUE THREAD

**Issue by be-next-hotdog** [pinguinjkeke/vue-local-storage#44]
Heyy, guy,
Quick question after a quick test: this lib support SSR mode now ? 
I got still error kind of `TypeError: Cannot read property 'set' of undefined`, any tips ? 

Thx for help.

**Comment by pinguinjkeke** [comment]
Yes, it does. It will not set up the Vue plugin when you're on SSR. But you mustn't call any of ls methods when you're inside server environment.
